### PR TITLE
Feature/basic people

### PR DESCRIPTION
**Pull request by philipkcl**
* implementation of person init form 
* implementation of person full form 
* create component Recref 
* a lot of refactoring for reusable modules
* move docker folder to `docker/site-edit-2`

### CODE REVIEW

File: core/helper/model_utils.py [hunk 1]
```diff
@@ -36,3 +39,34 @@ def create_eq_query(field, value):
...
+            init_val = 1
+
+        cursor.execute(f"CREATE SEQUENCE {seq_name} start with {init_val} ")
+        cursor.execute(nextval_sql)
+
+    result = cursor.fetchone()
+    return result[0]
+
+
+def default_current_timestamp():
```
Comment (J4bbi) [inline core/helper/model_utils.py:67]: why not set default=django.utils.timezone.now when you define the field?

File: person/forms.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                           ('F', 'Female'),
+                           ('', 'Unknown or not applicable'),
+                       ])
+                       )
+
+    is_organisation = form_utils.ZeroOneCheckboxField(required=False, initial='1', )
+    roles_titles = forms.CharField(required=False,
+                                   widget=forms.CheckboxSelectMultiple(
+                                       choices=[
+                                           ('5', 'Antiquary'),
```
Comment (J4bbi) [inline person/forms.py:36]: is it a good idea to hard-code these?
is it not in a table?

File: core/helper/model_utils.py (hunk 1 above)
Comment (philipkcl) [reply core/helper/model_utils.py:67]: yes, if django.utils.timezone.now is better, I will change it to
```
def default_current_timestamp():
   return django.utils.timezone.now()
```

Comment (J4bbi) [reply core/helper/model_utils.py:67]: sure, but why have the utility function at all, why not just put the method directly?

File: person/forms.py (hunk 2 above)
Comment (philipkcl) [reply person/forms.py:36]: For now should be fine, 
If other code need to use those choices, I will extract as constant variable.
If there have requirement that User can change those choice in website, I will extract to those values to DB

**Pull request merged**